test/controllers/media_instances_controller_test.rb

```diff
@@ -0,0 +1,216 @@
+require "test_helper"
+
+describe "MediaInstancesController" do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
+  describe "index" do
+    it "returns a success status for all media_instances" do
+      get media_instances_path
+      must_respond_with :success
+    end
+
+    it "returns a success status when there are no media_instances" do
+      MediaInstance.destroy_all
+      get media_instances_path
+      must_respond_with :success
+    end
+
+    it "returns a success status when given a valid user_id" do
+      get user_media_instances_path(user.first)
+      must_respond_with :success
```

Comment:
I don't understand what the `user_media_instances_path` is for, and I don't believe it's used anywhere in your application. This is usually a good indication that the route isn't needed.